platform/solr/solr-logging/pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>solr</artifactId>
+        <groupId>ddf.platform.solr</groupId>
+        <version>2.13.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>solr-logging</artifactId>
+    <name>DDF :: Platform :: Solr :: Logging Dependencies</name>
+    <description>Shaded jar of Solr logging dependencies.</description>
+    <packaging>jar</packaging>
+    <properties>
+        <org.slf4j.version>1.7.24</org.slf4j.version>
```

Comment:
❓ Why are we defining these here?

Reply:
These are specific to the version Solr requires.  They conflict with the versions used by Pax Logging and the rest of the system.